File: Other/Prime.py
from nose.tools import assert_equal

def prime(num):
	primes = [2, 3, 5, 7]
	count = 0
	if num in primes:
		return True
	for n in primes:
		if num%n == 0:
			count += 1
	if count == 0:
		return True
	else:
		return False

class Prime(object):
	def test(self, func):
		assert_equal(func(2), True)
		assert_equal(func(7), True)
		assert_equal(func(11), True)
		assert_equal(func(41), True)
		assert_equal(func(4), False)
		assert_equal(func(50), False)
		print("TESTS PASSED")

t = Prime()
t.test(prime)

File: Other/Prime.js
function prime(n){
	var count = 0;
	var nums = [2, 3, 5, 7];
	nums.forEach(function(num){
		if(n%num==0){
			count ++;
		}
	});
	if(count == 0 || n == 1 || n === 2 || n === 3 || n === 5 || n === 7){
		console.log("Is a prime");
	} else{
		console.log("Not a prime");
	}
}

prime(1);
prime(5);
prime(20);
prime(50);
prime(17);